README.md

```diff
@@ -5,12 +5,12 @@ just be moved to a new file with few problems, except for a few files that were
 up, for example the `lundump.c` file.
 
 In its current state the project does not aquire any additional CHERI security guarantees.
-However, this would allow for experimenting with different groupings of functions by linking the
+However, this will allow for experimenting with different groupings of functions by linking the
 different object files together in different ways. The main purpose of doing this is to get
 additional security under CHERI without any CHERI specific code. This can be done by exploiting
 the runtime linker of CheriBSD.
 
-Some of the Lua tests do not pass, and they are commented out. One of the popen tests in
+Some of the Lua tests do not pass, and they are commented out: One of the popen tests in 
```

Comment:
s/One/one/ (no capital letters after `:` :) )

Reply:
I didn't know about that.